typescript/packages/lattice-client-core/src/lattice-client.ts

```diff
@@ -16,10 +15,12 @@ type LatticeClientConfig = {
   wadmTopicPrefix?: string;
 };
 
+type LatticeConnectionFunction = (config: LatticeClientOptions['config']) => LatticeConnection;
```

Comment:
```suggestion
type LatticeConnectionCtor = (config: LatticeClientOptions['config']) => LatticeConnection;
```

This is essentially a constructor right? I'm not sure I *love* "Ctor", but there might be a more intuitive name for this